packages/terra-functional-testing/src/commands/utils/ScreenshotRequestor.js

```diff
@@ -117,15 +117,25 @@ class ScreenshotRequestor {
    * Downloads the screenshots and unzip it to the reference screenshot directory defined by referenceScreenshotsPath.
    */
   async downloadScreenshots() {
-    const archiveUrl = `${this.serviceUrl}/reference.zip`;
-    const response = await fetch(
-      archiveUrl,
-      {
+    var archiveUrl;
+    var fetchOptions;
```

Comment:
Is there a reason we are using `var` here rather than `let`?

Reply:
Because I'm dumb and forgot run lint before PRing.

Fixed here https://github.com/cerner/terra-toolkit/pull/808/commits/d3af3b622c891d1735c512997b87b823ecc3c8d6